IHolidayParser.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tavisca.Applause
+{
+    public interface IHolidayParser
+    {
+        List<Holiday> Parse(DataTable holidayDetails);

```

Comment:
As per the inputs, this looks like an extension method. Any specific reason for adding it as part of interface?